src/Tizen.NUI/src/public/Events/GestureOptions.cs

```diff
@@ -326,6 +327,16 @@ public void SetLongPressMinimumHoldingTime(uint time)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
-
+        /// <summary>
+        /// Sets the maximum allowed time required to be recognized as a multi tap gesture.
+        /// </summary>
+        /// <remarks>This is a global configuration option. Affects all gestures.</remarks>
+        /// <param name="time">The time value in milliseconds</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetTapMaximumAllowedTime(uint time)
```

Comment:
파라메터 이름을 ms로 바꾸면 문서를 보지 않더라도 milliseconds로 유추 가능할것 같습니다.

그리고 더블 탭으로 인식하기 위한 timeout시간을 설정하는것이 맞다면,
타 플랫폼에서도 사용하는 용어를 사용하는것은 어떨까요?
![image](https://user-images.githubusercontent.com/1029155/132209570-4d948dd6-4cb4-44f7-9053-765af2858b54.png)
https://developer.android.com/reference/android/view/ViewConfiguration#getDoubleTapTimeout()

```suggestion
        public void SetTapMaximumAllowedTime(uint ms)
```

Reply:
time은 ms로 변경하겠습니다.

탭과 탭 사이의 timeout 설정하는 api는 맞습니다.
그런데 꼭 더블 탭이 아니라 탭과 탭 사이의 time을 설정하는 것이라 (트리플 탭인 경우도 있음)
double tap timeout은 경우에 따라 맞지 않을 수 있을 것 같습니다.
혹시 이 경우에도 double tap timeout이란 용어를 사용하나요?

Reply:
네, 타 플랫폼도 동일한 의미로 double tap timeout을 사용하는것 같습니다.

Reply:
네 감사합니다. SetDoubleTapTimeout으로 이름 변경하였습니다.